invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -0,0 +1,189 @@
+    sources = fields.Method('get_sources')
+    languages = fields.Method('get_languages')
+    coverage = fields.Method('get_locations')
+    formats = fields.Method('get_formats')
+
+    def get_titles(self, obj):
+        """Get titles."""
+        return [obj['metadata']['title']]
+
+    def get_identifiers(self, obj):
+        """Get identifiers."""
+        items = []
+        items.extend(
+            i['identifier'] for i in obj['metadata'].get('identifiers', [])
+        )
+        items.extend(p['identifier'] for p in obj.get('pids', {}).values())
+
+        return items
+
+    def get_creators(self, obj):
+        """Get creators."""
+        return [
+            c['person_or_org']['name']
+            for c in obj['metadata'].get('creators', [])
+        ]
+
+    def get_relations(self, obj):
+        """Get relations."""
+        rels = []
+
+        # Fundings
+        fundings = obj['metadata'].get('funding', [])
+        for f in fundings:
+            award = f.get('award')
+            if award:
+                rels.append(
+                    u'{0}:{1}'.format(award['scheme'], award['identifier'])
+                )
+
+            funder = f.get('funder', {})
+            if funder:
+                rels.append(
+                    u'{0}:{1}'.format(funder['scheme'], funder['identifier'])
+                )
+
+        # Alternate identifiers
+        for a in obj['metadata'].get('alternate_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        # Related identifiers
+        for a in obj['metadata'].get('related_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        return rels
+
+    def get_rights(self, obj):
+        """Get rights."""
+        rights = []
+        for right in obj['metadata'].get('rights', []):
+            rights.append(right['title'])
+
+            license_url = right.get('link')
+            if license_url:
+                rights.append(license_url)
+
+        return rights
+
+    def get_dates(self, obj):
+        """Get dates."""
+        dates = [obj["metadata"]["publication_date"]]
+
+        if obj['access']['status'] == 'embargoed':
+            dates.append(obj['access']['embargo']['until'])
+
+        return dates or missing
+
+    def get_descriptions(self, obj):
+        """Get descriptions."""
+        metadata = obj['metadata']
+        descriptions = []
+
+        description = metadata.get("description")
+        if description:
+            descriptions.append(description)
+
+        additional_descriptions = metadata.get("additional_descriptions", [])
+        for add_desc in additional_descriptions:
+            descriptions.append(add_desc['description'])
+
+        return descriptions
+
+    def get_subjects(self, obj):
+        """Get subjects."""
+        metadata = obj['metadata']
+        subjects = []
+        subjects.extend(metadata.get('keywords', []))
```

Comment:
There is no attribute `keywords` in the [metadata schema](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/schemas/metadata.py#L406)